hil/cli.py

```diff
@@ -1,81 +1,27 @@
-"""This module implements the HIL command line tool."""
-from hil import config, server, migrations
-from hil.config import cfg
-from hil.commands.util import ensure_not_root
+#!/usr/bin/env python
 
-import inspect
-import json
-import os
-import requests
 import sys
-import urllib
+import os
 import schema
-import logging
-
 import pkg_resources
+import click
+import requests
 
-from functools import wraps
-
-from hil.client.client import Client, RequestsHTTPClient, KeystoneHTTPClient
+from hil.client.client import Client, RequestsHTTPClient
 from hil.client.base import FailedAPICallException
 from hil.errors import BadArgumentError
+from hil.commands.util import ensure_not_root
 
 
-logger = logging.getLogger(__name__)
-command_dict = {}
-usage_dict = {}
-MIN_PORT_NUMBER = 1
-MAX_PORT_NUMBER = 2**16 - 1
 VERSION = pkg_resources.require('hil')[0].version
 
 # An instance of HTTPClient, which will be used to make the request.
 http_client = None
 C = None
```

Comment:
I would say we should try out best to avoid the "simple-letter-variable-name" situation since you are currently working on building a new CLI. Unless there is a convention on global variable name on Python that I am not aware of at the moment ;)

Reply:
cool, will do that. I just copy pasted this from the old cli for now. But yeah, might as well clean it up while we are at it.

Reply:
exactly. 👍